DatePicker/DatePickerBase.js

```diff
@@ -164,6 +173,15 @@ const DatePickerBase = kind({
 		 */
 		rtl: PropTypes.bool,
 
+		/**
+		 * Disables 5-way spotlight from navigating into the component.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		spotlightDisabled: PropTypes.bool,
```

Comment:
Could you explain why did you add this prop?

Reply:
to disable 5-way navigation on the component. Focus state is not clearly visible on Gallium, but it is on Silicon.

Reply:
Oh... I see..